### ISSUE TITLE
1.8.3 support

### ISSUE THREAD

**Issue by Granik24** [Kohle/MelonSlap#5]
Is on the plan support 1.8.3 version? :)

**Comment by Kohle** [comment]
I've taken a bit of a hiatus from this project recently, but I plan to get back into it soon. Yes, I'll be updating the project to new versions as needed!

**Comment by Granik24** [comment]
Yes! Can't wait for that. We want to use your plugin at server "hryozivot.cz".
